submissions/api.py

```diff
@@ -373,6 +383,40 @@ def get_submissions(student_item_dict, limit=None):
     return SubmissionSerializer(submission_models, many=True).data
 
 
+def get_all_submissions(course_id, item_id, item_type, read_replica=True):
```

Comment:
The parameter is used in the tests – not sure whether there is any other use case.  (And I've changed the code to actually consider this parameter.)